jsonapi/unmarshal.go

```diff
@@ -173,6 +187,13 @@ func setDataIntoTarget(data *Data, target interface{}) error {
 		return err
 	}
 
+	if castedTargetWithLid, ok := target.(UnmarshalLocalIdentifier); ok {
+		if err := castedTargetWithLid.SetLID(data.LID); err != nil {
+			return err
+		}
+		castedTarget, _ = castedTargetWithLid.(UnmarshalIdentifier)
```

Comment:
I guess it's not needed, because taget is Pointer otherwise ID will be lost in line 190